images/diagnostics/Dockerfile

```diff
@@ -0,0 +1,9 @@
+#
+# OpenShift diagnostics image
+# Used by network diagnostics (oadm diagnostics NetworkCheck)
+#
+# The standard name for this image is openshift/diagnostics-deployer
+
+FROM docker.io/busybox
```

Comment:
Please change this to openshift/origin-base to address @danwinship and @eparis's concerns.